app/views/layouts/_header.html.erb

```diff
@@ -7,7 +7,7 @@
     </div>
     <div class="links">
       <% if current_user.present? %>
-        <div class="create-casebook" data-action="show-casebook-modal"><%= t('header.links.create-casebook') %></div>
+        <button type="button" class="create-casebook" data-action="show-casebook-modal"><%= t('header.links.create-casebook') %></button>
```

Comment:
Is there a reason you used `button` instead of `button_to`?

Reply:
Aha. Sorry, I'm not used to Rails' view helpers. Can you show me how to use `button_to` here?

Reply:
Nevermind, I should be able to figure it out. Fix forthcoming.

Reply:
Aha! Yes, there is a reason.  If this is the right `button_to` (https://apidock.com/rails/ActionView/Helpers/UrlHelper/button_to), it sounds ike `button_to` creates a whole form that, when you click on the button, submits the form. Here, we just need a button to trigger some javascript that opens the modal and performs related housekeeping: no HTTP request involved.

Reply:
Ah! Ok --- I was wrong.